app/views/upload.scala.html

```diff
@@ -1,6 +1,47 @@
 @()(implicit messages: Messages)
 
 @main(Messages("upload.header")) {
-    <h1 class="govuk-heading-xl">@Messages("upload.title")</h1>
-    <p>Placeholder page for upload</p>
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+        <span>
+            <a href="/dashboard" class="govuk-back-link">Back</a>
+            <span class="govuk-caption-xl">Step 3 of 3</span>
```

Comment:
We've got some code that lets us autogenerate these "Step X of Y" messages without having to update them everywhere when the total changes. See https://github.com/nationalarchives/tdr-transfer-frontend/blob/master/app/views/seriesDetails.scala.html#L13 for example.

Reply:
Now using `progressIndicator` to generate dynamic messages from `Messages`. Can you take a look and see if I've done it inline with the other pages?